sonaranalyzer-dotnet/build/rspec.ps1

```diff
@@ -164,6 +164,7 @@ function CreateStringResources() {
             $description = $description -replace '&amp;', '&'
             $description = $description -replace '&lt;', '<'
             $description = $description -replace '&gt;', '>'
+            $description = $description -replace '\\!', '!'
```

Comment:
That's a workaround for an error in resgen.exe that fails when it sees `\!`... That's present in only one rule and I am not sure if I will not break something else if I fix it...